src/components/pages/Notes/NoteGroupNav.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { connect } from "react-redux";
+import { values } from "lodash";
+import { NavLink, Route } from 'react-router-dom';
+
+import Button from "../../library/Button";
+
+
+function NoteGroupNav({ children, noteGroups, addGroup, deleteGroup, updateGroupName }) {
+  const [editedGroupId, setEditedGroupId] = useState();
```

Comment:
👏 rather initialize the value to `null`. `null` is the explicit Javascript value that says that the value is empty.